objects/endpoint.rb

```diff
@@ -79,7 +79,9 @@ def to_h
       log: @item['log']
     }
     h[:updated] = Time.at(@item['updated']) if @item['updated']
-    h[:flipped] = Time.at(@item['flipped']) if @item['flipped']
+    # TODO: #75 consider other implementations of the null Time object that may
+    # be better than Time.new(0).
+    h[:flipped] = @item['flipped'] ? Time.at(@item['flipped']) : Time.new(0)
```

Comment:
@yegor256, my purpose was to follow [your advice on null reference](http://www.yegor256.com/2014/05/13/why-null-is-bad.html) and use sort of a null object pattern so we don't have to check for nil everytime we access 'flipped'.